dodam-student/src/main/kotlin/com/b1nd/dodam/student/DodamApp.kt

```diff
@@ -97,6 +99,10 @@ fun DodamApp(isLogin: Boolean, navController: NavHostController = rememberNavCon
         outingScreen(
             onAddOutingClick = { TODO("navigate to add outing screen") },
         )
+        wakeupSongScreen(
+            onAddWakeupSongClick = { TODO("navigate to add wakeup song screen") },
+            popBackStack = { navController.popBackStack() },
```

Comment:
Changing navController.popBackStack() to navController::popBackStack would make the code cleaner, I think.